.php_cs.dist

```diff
@@ -31,6 +31,7 @@ $rules = [
     'no_php4_constructor' => true,
     'no_useless_else' => true,
     'no_useless_return' => true,
+    'nullable_type_declaration_for_default_null_value' => true,
```

Comment:
```suggestion
    'nullable_type_declaration_for_default_null_value' => ['use_nullable_type_declaration' => true],
```

Reply:
@phansys Is `'nullable_type_declaration_for_default_null_value' => true,` invalid or are you asking for this change to avoid BC-break if the default value change ?

Reply:
The 2nd reason, plus I think we should be as verbose as possible here.